tests/test_InstHR.py

```diff
@@ -9,6 +9,7 @@
 myPath = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, myPath + '/../')
 from InstHR import ihr
+from HeartRateMonitor import HRM
 
 
 f = open('dummyEKGdata.csv', 'rU')
```

Comment:
you may want to factor this block out into a modular `setup` function that takes a filename and sets up an `HRM` object for testing. You can call this setup function with different files and params from each of your tests as needed.